packages/fluent-theme/src/fluent/styles/CommandBar.styles.ts

```diff
@@ -1,11 +1,15 @@
 import { ICommandBarStyleProps, ICommandBarStyles } from 'office-ui-fabric-react/lib/CommandBar';
 
 export const CommandBarStyles = (props: ICommandBarStyleProps): Partial<ICommandBarStyles> => {
+  const { theme } = props;
+  const { palette } = theme;
+
   return {
     root: [
       {
         height: 44,
-        padding: '0 14px 0 24px'
+        padding: '0 14px 0 24px',
+        backgroundColor: palette.white
```

Comment:
Maybe 🤔 make it transparent so that it can pick up the background color? Same goes for the `CommandBarButton`

Reply:
Checked with design on this, we want to keep it white